### PR TITLE
1.9 screenshots

### PR DESCRIPTION
**Pull request by joel-hamill**
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by brndnmtthws** [comment]
Looks good to me. If I'm going to be nit picky though, I'd suggest you crop out the browser window elements, and just include the web pages.

**Comment by joel-hamill** [comment]
@ashenden re: browser screenshots of the UI - do you think it's best to crop out the browser window entirely? i think it provides context for people as far as where they'll be running the UI.

**Comment by ashenden** [comment]
@joel-hamill do you have an example of what you're proposing or the options?

**Comment by joel-hamill** [comment]
@ashenden currently most of the screenshots look like this: https://github.com/dcos/dcos-docs/pull/825/files#diff-ffa16378843b8bed97004b56b0015363

**Comment by ashenden** [comment]
If we omit browser chrome, we should fit the window exactly.  Do not crop into the bounds of the window, or show extra pixels out of frame.  For example: https://cl.ly/0C223Y2C3I1E

If we wish to show the browser, we should frame it nicely on a neutral background, centered with even margins all around, strip out all browser chrome, and hide the dock and other toolbar shit.  For example: https://cl.ly/0C1E2m000504

Alternatively, you can use CMD+SHIFT+4 then space-bar to take a picture of the window with a shadow included.  This looks pretty nice too (https://cl.ly/3g2g0e1t1g05).

All 3 work for me as long as we are consistent.

**Comment by joel-hamill** [comment]
@ashenden k i'll file a ticket to make the shots consistent for 1.10: https://mesosphere.atlassian.net/browse/DOCS-1612

File: 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ To demonstrate the vhost feature:
 
 1.  Find your [public agent IP](/docs/1.9/administration/locate-public-agent/). 
 
-1.  Modify the external nginx app (`nginx-external.json`) to point to your public agent IP. You can modify your app by using the DC/OS CLI or GUI.
+1.  Modify the external nginx app (`nginx-external.json`) to point to your public DNS name. You can modify your app by using the DC/OS CLI or GUI.
```
Comment (emanic) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: Is it possible that this is the "fully qualified domain name (FQDN)"? I believe this is a more standard term.

File: 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md [hunk 2]
```diff
@@ -241,21 +242,19 @@ To demonstrate the vhost feature:
...
+    1.  Navigate to the **Services > Services > nginx-external** service and click edit.
  
-    1.  Select **Edit** > **Labels** > **Add Label**.
+    1.  Select **Environment > ADD LABEL**.
+    
+    1.  Enter `HAPROXY_0_VHOST` for **KEY** and specify your public DNS name for **VALUE**.
     
         ![Update app](/docs/1.9/usage/service-discovery/marathon-lb/img/nginx-external-gui.png)
         
+        **Important:** Do not include the leading `http://`trailing slash (`/`) in your public DNS name.
```
Comment (emanic) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: If we were calling it an FQDN, they could Google FQDN and discover this, as well.

**Comment by emanic** [comment]
Some conflicts @joel-hamill

File: 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md (hunk 1 above)
Comment (joel-hamill) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: @karlkfi thoughts ^?

File: 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ To demonstrate the vhost feature:
 
 1.  Find your [public agent IP](/docs/1.9/administration/locate-public-agent/). 
 
-1.  Modify the external NGINX app (`nginx-external.json`) to point to your public agent IP. You can modify your app by using the DC/OS CLI or GUI.
+1.  Modify the external nginx app (`nginx-external.json`) to point to your public DNS name. You can modify your app by using the DC/OS CLI or GUI.
```
Comment (karlkfi) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: "NGINX" is the official name, all caps.

**Comment by karlkfi** [comment]
On the screenshot topic, I would like screenshots of the GUI to include the path to the page, if possible. One way to do this is by showing it in the browser bar. Another way is to put it in the image caption. I'm nto sure which is better, but I like knowing what I'm looking at, and I know these screenshots often get pulled out and used in presentations. The downside of showing the URL bar is that the domain either needs to be masked (image editing) or manually replaced with a generic one like `dcos-example.com` before taking the screenshot (easier).

File: 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md (hunk 3 above)
Comment (karlkfi) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: I'm pretty sure there was recently a conversation about whether to use "public domain name", FQDN, "public ip", "public DNS name", "master IP", etc. I don't remember seeing an outcome. But it's a larger unresolved topic than just this PR.

Comment (joel-hamill) [inline 1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md]: Going to leave as `public agent DNS name` for now.

**Pull request merged**